src/components/Layout/index.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+Header
+Top/Bottom banner
+Nav
+Hero
+Main
+Content
+Left/Right Panel
+Left/right Sidebar
+Footer
+ */
+
+import { FC } from 'react'
+
+import { Layout as Root, type LayoutProps } from './Layout'
+import { LayoutHeader } from './LayoutHeader'
+import { LayoutContent } from './LayoutContent'
+import { LayoutHero } from './LayoutHero'
+import { LayoutFooter } from './LayoutFooter'
+import { LayoutTopBanner } from './LayoutTopBanner'
+import { LayoutBottomBanner } from './LayoutBottomBanner'
+import { LayoutLeadingSidebar } from './LayoutLeadingSidebar'
+import { LayoutTrailingSidebar } from './LayoutTrailingSidebar'
+import { LayoutLeadingPanel } from './LayoutLeadingPanel'
+import { LayoutTrailingPanel } from './LayoutTrailingPanel'
+
+export const Layout: FC<LayoutProps> & {
```

Comment:
IMO using the `.` pattern here is not good since it breaks code splitting

Reply:
🤷‍♂️